src/index.html

```diff
@@ -1,12 +1,373 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin = "anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/reset.css">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body body">
+    <header class="body__header header">
+      <a href="#home">
+        <img
+             src="./images/logo.png"
+             alt="Moyo"
+             class="header__logo"
+        >
```

Comment:
```suggestion
        <img
          src="./images/logo.png"
          alt="Moyo"
          class="header__logo"
        >
```
Pay attention to formatting